client/src/components/Header/Header.component.tsx

```diff
@@ -166,7 +162,7 @@ const Header = () => {
     }
   }, [matchQuestions?.pathname])
 
-  const expandedSearch = () => {
+  const ExpandedSearch = () => {
```

Comment:
Just curious, why is there a difference in nomenclature for `ExpandedSearch` (title case) compared to components like `websiteLinks` (camel case)?

Reply:
I missed `websiteLinks` and `authLinks`, so let me go fix those. Thanks for the catch!